### DISCUSSION TITLE
Crafting an Accessible and Inclusive Resumé

### DISCUSSION THREAD

**Discussion by streats** [community/community#62531] [Discover: GitHub Best Practices]
# Crafting an Inclusive and Accessible Resumé

Last year, during the job search that eventually brought me to GitHub, I revamped my resumé (also known as CV) to be more accessible. I’ve learned a lot about accessibility in the past couple of years, and some of my work as a content designer in technology has even included writing guidance on how to make content and documents more accessible. 

Realizing my resumé didn’t follow the principles I was applying to my work, I updated it to be simpler, easier to read, and more compatible with assistive technology. 

## Why is it important to make CVs accessible?
* Recruiters and hiring managers with disabilities exist, so it's important to be inclusive not just of our users and customers, but of our future colleagues too (who, in this case, are “users” of our CV!)
* Content that’s accessible is more usable for everyone. So even if the people reviewing your application can use PDFs just fine, they can still benefit from your resumé being clear and simple.
* Making your resumé accessible is also a great way to demonstrate you know how to make content accessible. Whether you’re applying for a role in content, design, development, or literally any other job that involves written communication, content accessibility is a great skill to be able to not just say you have, but show you have.

Applying some of the basic principles I’ve picked up on the job, here are some of the things I now do to improve the accessibility of my CV. 

## Avoid using tables to create the layout
Previously, I used a single-row table to arrange job details across the width of the page. The company and title were in the left column, the job dates in the right column, and the description was listed underneath the table, in the normal body text of the page. 

<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/github/community-engagement/assets/12902836/cb094b98-36a5-4017-99ec-308f8b9e8394">
  <source media="(prefers-color-scheme: light)" srcset="https://github.com/github/community-engagement/assets/12902836/045f01c9-1f27-41ac-9288-c844e9e102af">
  <img alt="Single-row table with two columns. The first cell says 'Senior Content Designer, GitHub' and the second says 'Since June 2022'. Underneath the table it says 'Content discovery and design for GitHub Docs team to ensure documentation sets meet user needs, are scalable, discoverable and accessible.'" src="https://github.com/github/community-engagement/assets/12902836/045f01c9-1f27-41ac-9288-c844e9e102af">
</picture>

This layout makes good use of the horizontal space on the page, but it can be distracting, confusing and even uncomfortable by leading the eye in multiple directions with no clear reading order. In addition, tables that aren’t properly marked up with a header row and relationships between rows and columns, or tables with merged cells, can be difficult for screen readers to make sense of.

I removed these tables and now write information in a normal linear flow:

* Job title, Company
* Dates
* Description

This of course can mean the content runs over more pages, but I think this is a case where accessibility trumps brevity - or rather, the illusion of brevity, since it’s the same amount of information, just presented differently. Three pages of well laid out, easy to read information is better than two pages of squished together text in small font with very little whitespace. Giving content more breathing space can also make it more accessible for people who struggle with high word density ("walls of text") such as people with ADHD or some visual disabilities. 

This also encourages me to be discerning about what information I really need to include in my resumé.

## Use proper header formatting (H1 through H4)
Instead of using bold formatting or manual font size changes to separate each section of my resumé, I use properly marked up header styles to make the structure of the information readable for assistive technology. 

I also find it a much more efficient way to apply consistent styling and content hierarchy. Since I use this method when designing content in my work, I'm used to using keyboard shortcuts to quickly apply these styles in Google Docs or Markdown files. 

In Google Docs, this looks like the following:

1. `Heading 1` Annie Streater
   1. `Heading 2` Professional experience
      1. `Heading 3` Senior Content Designer, GitHub
         1. `Heading 4` Since June 2022
            1. `Normal text` Content discovery and design for GitHub Docs team to ensure documentation sets meet user needs, are scalable, discoverable and accessible. 
      1. `Heading 3` Technical Writer, UK Government Cabinet Office
         1. `Heading 4` October 2020 to May 2022
            1. `Normal text` Content design, research and strategy for central government. Developing guidelines on digital, data and technology for government departments and public sector organizations.
   1. `Heading 2` Skills
   1. `Heading 2` Education and training

The [Document Outline tool in Google Docs](https://support.google.com/docs/answer/6367684?sjid=3139580512359981728-EU) shows you an outline of your content based on the heading styles. It can be helpful to use this to check the hierarchy of your resume is properly structured. 

In Markdown, using structured headings looks like the following:

```
# Annie Streater

## Professional experience

### Senior Content Designer, GitHub
#### Since June 2022
Content discovery and design for GitHub Docs team to ensure documentation sets meet user needs, are scalable, discoverable and accessible. 

### Technical Writer, UK Government Cabinet Office
#### October 2020 to May 2022
Designing guidance for digital teams in government to help them build citizen-centered services. 
```
## Make everything left-aligned
Previously I had my personal information - my name, contact information and job title - centered at the top of my resumé. Some content, like job dates, was right-aligned, and the rest of the information was left-aligned. 

<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/github/community-engagement/assets/12902836/20c1a1bb-a63e-4718-952f-7393bcb0db64">
  <source media="(prefers-color-scheme: light)" srcset="https://github.com/github/community-engagement/assets/12902836/7d6e3ffc-926b-41ca-8b58-71933902aed7">
  <img alt="Screenshot of @streats's resumé in Google Docs, with contact links, name and job title center-aligned at the top of the page. Underneath, the job title 'Senior Content Designer, GitHub' is left-aligned, and on the same row of text, the employment date 'Since June 2022' is right-aligned. Underneath, the job description says 'Content discovery and design for GitHub Docs team to ensure documentation sets meet user needs, are scalable, discoverable and accessible.' and is left-aligned, spanning the width of the page." src="https://github.com/github/community-engagement/assets/12902836/7d6e3ffc-926b-41ca-8b58-71933902aed7">
</picture>

Similar to the problems caused by using a table to distribute text across the width of the page, using multiple text alignment can be distracting for the eye. I now have everything left-aligned, which in combination with the linear structure, makes everything flow much more easily for reading left to right, top to bottom. 

<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/github/community-engagement/assets/12902836/0eb5bfb3-6492-48cf-bad3-b6f611b6b879">
  <source media="(prefers-color-scheme: light)" srcset="https://github.com/github/community-engagement/assets/12902836/a5162f42-2491-4adb-b57d-a05c45e6fc20">
  <img alt="Screenshot of @streats's resumé in Google Docs. All content is left-aligned." src="https://github.com/github/community-engagement/assets/12902836/a5162f42-2491-4adb-b57d-a05c45e6fc20">
</picture>

## Avoid abbreviations that can make content less readable
When trying to fit content in a certain amount of space, like the standard-recommended two pages, we often take shortcuts to abbreviate information. For example, using “etc”, “eg.” or using a slash (“/”) to straddle similar concepts. While using these notations can use fewer characters, this can often come at the cost of legibility or clarity. 

Screen reader software often reads out “e.g.” as “egg” if you don’t include the periods, or “apps/websites” as “apps websites”, which might confuse non-visual readers. Latin-derived notations like “e.g.” and “etc.” might also be confusing or unclear for people with lower levels of English literacy. For example, some people who use English as a Foreign Language, are Deaf, or have a learning disability may not be familiar with these terms. 

Making sure content is clear and readable is particularly helpful to make it accessible for:
* people with learning disabilities such as dyslexia
* neurodiverse people such as autistic people or people with ADHD
* people who struggle with "brain fog", such as people with depression or those recovering from a brain injury

Writing things out fully in my CV (“for example”, “apps and websites”) sometimes means a sentence awkwardly carries over to a new line, or a bullet point becomes just a bit too long. This pushes me to find creative ways to rewrite things more succinctly while still maintaining clarity and readability. It also challenges me to be clear about what I’m really trying to communicate - do I mean “apps or websites” or “apps and websites?”

In addition, I’ve found these abbreviations don’t add much value and can actually make my profile seem more generic. Consider the difference between these two sentences: 

* Experience in user research eg interviews, testing etc
* Experience in user research: interviews, usability studies, highlighter testing, prototyping.

Which one sounds more professional and confident?

## Make links descriptive
My CV generally only includes a couple of links - my email address, LinkedIn profile, and GitHub profile. Sometimes, I also provide a link to my portfolio or specific examples of my writing relevant to the role I’m applying for. In addition, I’ve recently started including a link to a version of this blogpost at the end of my CV, to let people know how and why I’ve made my CV more accessible.

Making links accessible generally means making hyperlinked text descriptive, so people know where the link goes and what it’s about. So a sentence saying “Click here to see my portfolio”, with only the word “here” hyperlinked, wouldn’t be particularly accessible for people who use assistive technology to navigate through a list of links in isolation. 

Instead, I make sure any hyperlinked text is descriptive: “For examples of my work, view my portfolio”, with “view my portfolio” as the link text. For my email address, I write it out in full and make sure it’s set as an email link (Google Docs does this automatically). This means people can use it as a link directly or copy and paste it as they need (for example, to enter into an applicant tracking system or interview schedule).

## Share in HTML or as an open format document
Although PDFs are the most common file format for sharing and uploading resumés, they’re not great for accessibility. Since they’re designed to be used for printed material, PDFs don’t work well on mobile devices or for people who use screen magnification, as the text doesn’t rearrange to fit the screen. This means people have to scroll in multiple directions to read all the information. 

While it is possible to mark up PDFs to work with screen readers and other assistive technology, this is quite technical work and some job application systems strip this information out. In addition, some commercial word processing file formats like .doc or .pages can cause strange formatting issues if people use different software to open the file. 

Now, where possible, I share or upload my resumé in:

* HTML (web page), which presents content as "view only" but is more compatible with different devices and settings.
* DOCX, a standard word processing document file type that’s similar to a Word `.doc` file but is “open format” meaning you can create and read them even if you don’t have Microsoft software.
* ODT, another "open format" text file similar to DOCX, though this format is often not accepted by job application platforms. 

## What have I learned?
By making my resumé more accessible, I’ve pushed myself to think differently. I’ve learned to be more concise and strategic with the content of my CV, and have had to challenge some of my own assumptions about technology and about the hiring process in general.

I’ll likely never know if it’s made a difference to how people have viewed and read my resumé, but I believe it was a worthwhile exercise. And while I hope I won’t be looking for a job any time soon, I’m definitely going to continue applying these principles and encouraging others to consider doing the same.  

## Try it yourself
Whether or not you're currently looking for a new role, it's a good idea to regularly update your resumé with your most recent accomplishments. So why not take the opportunity to revamp your resumé using these tips to make it more accessible? You can also apply many of these principles to your LinkedIn and GitHub profiles, and any other professional online presence you have.

**Comment by prestel01** [comment]
Good day, I need support

**Comment by rmonet1203** [comment]
hello

**Reply by Pedrovct2** [reply]
Oii

**Reply by Pedrovct2** [reply]
![Uploading 1663292974225.jpg…]()

**Comment by mubasher117** [comment]
Amazing!

**Comment by rubelislam788** [comment]
Hhh

**Comment by cpauvert** [comment]
Hi @streats,
I appreciated reading your experience regarding accessible resume, thank you for sharing. I was wondering what was your thoughts regarding data-driven resumés in standard format and especially how they could change accessibility, hopefully for the better.

For example, I recently stumble upon the JSON Resume initiative ([view their Github organization](https://github.com/jsonresume)) and noted several platforms, like [resumake](https://resumake.io/) or [ResumeFodder](https://www.resumefodder.com/), that propose conversion of such JSON to PDF or DOC. How do you think the community can help make these standards even more accessible?

Best regards, 

PS: I am not affiliated to any of the links I sent in this message.

**Reply by streats** [reply]
Hi @cpauvert, thanks for your thoughtful and interesting comment! Here's my personal view:

I haven't heard of data-driven resumés before but I certainly think that anything that can help people easily follow best practices is worth exploring. Standardized data structures seem like they could offer lots of potential to make sure resumés follow a consistent model (for example, ensuring correct header structure and content hierarchy), while also providing the flexibility to output to different formats. From an accessibility perspective, interoperability and cross-compatibility are really important (since not everyone has the same software and not all file types are well-supported by assistive technology), so being able to easily convert or export your resumé into different file types would be a good thing, as long as those file types were appropriate. 

As I mentioned in the post, PDFs can be tricky to make accessible, so I would still avoid using this format where possible. However, converting JSONs to [OpenDocument](https://en.wikipedia.org/wiki/OpenDocument), HTML and other open formats has the benefit of being both more accessible (in both technical and financial terms), and supported by most job platforms and applicant tracking systems. I would love to see these data-driven resumé platforms include more of these formats. At the same time, there's change that needs to happen in the recruitment industry too, as PDF is still by far the most used format, so it's unsurprising that resumé tools primarily focus on this. 

In terms of how such initiatives and approaches might help improve the accessibility landscape, ultimately I think anything we can do to promote openness, flexibility and choice will help make the web more accessible, as well as hopefully bringing accessibility front of mind. When designing or developing something - whether code-based or otherwise - following [core accessibility principles to ensure things are Perceivable, Operable, Understandable, and Robust](https://www.w3.org/WAI/fundamentals/accessibility-principles/) is generally going to be a great starting point. 

Thanks again for sharing!

**Reply by cpauvert** [reply]
I was happy to read your answer @streats. Thank you for taking this time. You are right, every small steps counts when we keep openness and accessibility in mind, as well as a cultural change that can only take time to percolate.

PS: I just realize that the link to [the website Resumefodder.com is down for everyone](https://downforeveryoneorjustme.com/resumefodder.com). Let's hope the project is still active.

**Comment by rhino701** [comment]
Thanks

**Comment by Anote-web** [comment]
LoL

**Comment by Alexei93-93** [comment]
Добрый вечер, занятная научная статья 👍 надеюсь, с ее помощью я смогу сдвинуть груз, который постоянно мне мешает идти по жизни. Спасибо автору, ты легко читаешь и информация изложенная тобою впитывается как вода👍

P.S. не туда оставил коммент первый раз, но будем исправляться 😁 Спасибо еще раз
